src/sdam/server.ts

```diff
@@ -62,6 +63,8 @@ const stateTransition = makeStateMachine({
   [STATE_CLOSING]: [STATE_CLOSING, STATE_CLOSED]
 });
 
+const SERVER_CLOSED_ERROR = 'Server is closed';
```

Comment:
In general, I think that when we have a resource that has been closed, the message should be of the form 

```javascript
'<resource type> has been closed'
```
That's a personal preference, but I think having it be more consistent would be better for error message readability. I'll flag other instances of this.

Reply:
I'm a little hesitant to use the past tense in this error message because it might imply that the server has been closed as a result of this operation rather than conveying the current state of the server. Happy to see what everyone else thinks; maybe I'm being a little _too_ semantic.